test/integration/backward_compatible/statistics/StatisticsTest.js

```diff
@@ -97,7 +97,7 @@ describe('StatisticsTest (default period)', function () {
 
     function contains(base, search) {
         const firstIndex = base.indexOf(search);
-        return firstIndex > -1 && firstIndex == base.lastIndexOf(search);
+        return firstIndex > -1 && firstIndex === base.lastIndexOf(search);
```

Comment:
Looking at this code, I feel like its name can be containsOnce. We can do this change later(not in this pr) though

Reply:
Yes, let's do it later